go.mod

```diff
@@ -1,8 +1,6 @@
 module zotregistry.dev/zot
 
-go 1.22.0
-
-toolchain go1.22.3
+go 1.22.5
```

Comment:
Do we really need to update the go directive?
This is the minimum version supported for building zot. Older versions would fail the build.